libs/test/SmartSyncTest/src/com/salesforce/androidsdk/smartsync/target/ParentChildrenSyncTest.java

```diff
@@ -248,38 +255,10 @@ public void testGetDirtyAndNonDirtyRecordIds() throws JSONException {
         cleanRecord(CONTACTS_SOUP, mapAccountToContacts.get(accounts[5])[0]);
 
         // Only clean account with clean contacts should not be returned
-        tryGetDirtyRecordIds(new JSONObject[] { accounts[0], accounts[1], accounts[2], accounts[4], accounts[5]});
+        tryGetDirtyRecordIds(new JSONObject[]{accounts[0], accounts[1], accounts[2], accounts[4], accounts[5]});
 
         // Only clean account with clean contacts should be returned
-        tryGetNonDirtyRecordIds(new JSONObject[] { accounts[3] });
-    }
-
-    /**
-     * Test deleteRecordsFromLocalStore with a master-detail relationship (children should be deleted too)
```

Comment:
The target method was changed (to not automatically delete children) so those tests are no longer relevant.